include/aspect/velocity_boundary_conditions/gplates.h

```diff
@@ -333,55 +267,66 @@ namespace aspect
 
       private:
         /**
-         * A variable that stores the current time of the simulation, but
-         * relative to the velocity_file_start_time.
+         * A variable that stores the currently used data file of a series. It
+         * gets updated if necessary by update().
          */
-        double time_relative_to_vel_file_start_time;
+        int  current_file_number;
 
         /**
-         * A variable that stores the currently used velocity file of a
-         * series. It gets updated if necessary by set_current_time.
+         * Time from which on the data file with number 'First data file
+         * number' is used as boundary condition. Previous to this time, 0 is
+         * returned for every field. Depending on the setting of the global
+         * 'Use years in output instead of seconds' flag in the input file,
+         * this number is either interpreted as seconds or as years."
          */
-        int  current_time_step;
+        double first_data_file_model_time;
 
         /**
-         * Time at which the velocity file with number 0 shall be loaded.
-         * Previous to this time, a no-slip boundary condition is assumed.
+         * Number of the first data file to be loaded when the model time is
+         * larger than 'First data file model time'.
          */
-        double velocity_file_start_time;
+        int first_data_file_number;
 
         /**
-         * Directory in which the gplates velocity are present.
+         * In some cases the boundary files are not numbered in increasing but
+         * in decreasing order (e.g. 'Ma BP'). If this flag is set to 'True'
+         * the plugin will first load the file with the number 'First data
+         * file number' and decrease the file number during the model run.
          */
-        std::string data_directory;
-
-        /**
-         * First part of filename of velocity files. The files have to have
-         * the pattern velocity_file_name.n.gpml where n is the number of the
-         * current timestep (starts from 0).
-         */
-        std::string velocity_file_name;
+        bool decreasing_file_order;
 
         /**
          * Time in model units (depends on other model inputs) between two
-         * velocity files.
+         * data files.
```

Comment:
I think this was my fault copying the stuff from ascii data. Could you replace this with velocity again?